data/json/snippets/tutorial.json

```diff
@@ -157,7 +157,7 @@
       },
       {
         "id": "LESSON_RESTORE_STAMINA",
-        "text": "Bashing things is hard work that will deplete your stamina.  You can see how much stamina you have on the sidebar.  The less stamina you have, the slower your movement and most actions will be.  To restore stamina, you need only to wait.  You can press <keybind:wait> to open the <color_light_cyan>Wait for how long</color> menu and select the amount of time you want to wait.  Select option <color_light_cyan>Wait until you catch your breath</color> to fully restore all lost stamina.\n\nBe careful about doing too many strenuous activities as you will build up your weariness over time, which will stack movement and action penalties on top of your stamina cost.  You can watch your <color_light_cyan>Weariness Transition</color> in the sidebar as you are performing longer activities to see their effect on  your weariness.  To reduce your weariness, you just need to rest or do less strenuous activities for a while."
+        "text": "Bashing things is hard work that will deplete your stamina.  You can see how much stamina you have on the sidebar.  The less stamina you have, the slower your movement and most actions will be.  To restore stamina, you need only to wait.  You can press <keybind:wait> to open the <color_light_cyan>Wait for how long</color> menu and select the amount of time you want to wait.  Select option <color_light_cyan>Wait until you catch your breath</color> to fully restore all lost stamina.\n\nBe careful about doing too many strenuous activities as you will build up your weariness over time, which will stack movement and action penalties on top of your stamina cost.  You can watch your <color_light_cyan>Weary Transition</color> in the sidebar as you are performing longer activities to see their effect on  your weariness.  To reduce your weariness, you just need to rest or do less strenuous activities for a while."
```

Comment:
I could have sworn it said Weariness, but maybe my brain autofilled it in to sound nicer. Good catch.